compose/koin-compose/src/commonMain/kotlin/org/koin/compose/KoinApplication.kt

```diff
@@ -61,9 +61,8 @@ private fun getDefaultKoinContext() = KoinPlatformTools.defaultContext().get()
  * @author @author jjkester
  */
 @Composable
-fun getKoin(): Koin = currentComposer.run {
-    return LocalKoinApplication.current
-}
```

Comment:
currentComposer.run is unnecessary

Reply:
yeah 🤔 I would need to benchmark this, being sure we don't miss something behind the scene

Reply:
@arnaudgiuliani 
From AndroidX:

https://github.com/androidx/androidx/blob/6d1f16a70c802850a83daae8929ea1662f972715/compose/runtime/runtime/src/commonMain/kotlin/androidx/compose/runtime/CompositionLocal.kt#L69-L75

```kotlin
@Stable
sealed class CompositionLocal<T> constructor(defaultFactory: () -> T) {
    internal val defaultValueHolder = LazyValueHolder(defaultFactory)

    internal abstract fun updatedStateOf(value: T, previous: State<T>?): State<T>

    /**
     * Return the value provided by the nearest [CompositionLocalProvider] component that invokes, directly or
     * indirectly, the composable function that uses this property.
     *
     * @sample androidx.compose.runtime.samples.consumeCompositionLocal
     */
    @OptIn(InternalComposeApi::class)
    inline val current: T
        @ReadOnlyComposable
        @Composable
        get() = currentComposer.consume(this)
}
```

Reply:
add `@ReadOnlyComposable` to `getKoin()` will optimize the generated code.
Docs: https://developer.android.com/reference/kotlin/androidx/compose/runtime/ReadOnlyComposable

Reply:
thanks

Reply:
I have just fixed the conflict. Could you check this PR again 🙏 ?